check_k8s.py

```diff
@@ -78,5 +93,14 @@ def main():
     sys.exit(output.state.value)
 
 
+def is_ignored_resource(line, expressions):
+    word = re.search(r"\S+:{1}", line)
+    for expression in expressions:
+        found = re.search(expression, word.group())
+        if found:
+            return True
```

Comment:
We should debug log matches so it's easier for customers to know what they did wrong.